Extensions/alternate_audio_player.js

```diff
@@ -0,0 +1,60 @@
+//* TITLE Alternate Audio Player **//
+//* VERSION 0.1.0 **//
+//* DESCRIPTION	Adds an alternative (working) audio player to audio posts. **//
+//* DETAILS This extension adds a native HTML5 audio player to every audio post, which should always work as well as enabling looping, downloading, and better seeking. **//
+//* DEVELOPER akunohomu **//
+//* FRAME false **//
+//* BETA true **//
+
+XKit.extensions.alternate_audio_player = new Object({
+
+	running: false,
+
+	preferences: {
+		"default_volume": {
+			text: "Default volume",
+			default: 1.0,
+			value: 1.0,
+			type: "combo",
+			values: [
+				"100%", 1.0,
+				"75%", 0.75,
+				"50%", 0.50,
+				"25%", 0.25
+			],
+			desktop_only: true
```

Comment:
Yeah, I feel that, but right now we don't have a way of indicating that (I
think) so it's probably best to leave this out for now, because it's just
confusing code to read.

On Mon, Aug 8, 2016 at 5:37 PM Aku no Homu <notifications@github.com> wrote:

> In Extensions/alternate_audio_player.js
> <https://github.com/new-xkit/XKit/pull/1126#discussion_r73960033>:
>
> > +
> > +	running: false,
> > +
> > +	preferences: {
> > +		"default_volume": {
> > +			text: "Default volume",
> > +			default: 1.0,
> > +			value: 1.0,
> > +			type: "combo",
> > +			values: [
> > +				"100%", 1.0,
> > +				"75%", 0.75,
> > +				"50%", 0.50,
> > +				"25%", 0.25
> > +			],
> > +			desktop_only: true
>
> Confusion. But this entire extension is unnecessary on the mobile site -
> Tumblr already uses a native player there.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/new-xkit/XKit/pull/1126/files/f09e942578147482af53dc37025892889bdcae18#r73960033>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAORV7F7l0ewLSNkFMa4BgufUDBcTW96ks5qd6F8gaJpZM4JeaiX>
> .
>